virttest/utils_misc.py

```diff
@@ -41,12 +47,14 @@
 from avocado.utils import linux_modules
 from avocado.utils import memory
 from avocado.utils.astring import string_safe_encode
+from avocado.utils.process import CmdResult
 
 from . import data_dir
 from . import error_context
 from . import cartesian_config
 from . import utils_selinux
 from . import utils_disk
+from . import logging_manager
```

Comment:
I think we need to change all relative path to absolute path? @chunfuwen

Reply:
@czhang0 ,please take this opportunities to replace all relative imports in this file

Reply:
Ok, I will use absolute import here.